code/game/gamemodes/gang/gang.dm

```diff
@@ -228,17 +232,15 @@ GLOBAL_VAR_INIT(deaths_during_shift, 0)
 				new_wanted_level = 4
 			if(FIVE_STARS_LOW to INFINITY)
 				new_wanted_level = 5
-	if(wanted_level == new_wanted_level) //Same shit, dont care.
```

Comment:
why remove this

Reply:
it caused stars to not appear until a star jump to level 2 occured

Reply:
new converts also couldn't see stars

Reply:
then that's a bug with the icon creation id imagine, this isn't the right fix I think

Reply:
I'll take a peek tommorow

Reply:
aight but this'll get merged in the mean time because of the announcement spam